dev-docs/bidders/apester.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: bidder
+title: Apester
+description: Apester Bidder Adaptor
+biddercode: apester
+pbjs: true
+pbs: false
+media_types: video, banner
+userIds: all
+fpd_supported: false
+gdpr_supported: true
```

Comment:
There's no `gvl_id` and I couldn't find any consent handling in your adapter. How do you support GDPR?

Reply:
Updated docs, thanks